src/state.rs

```diff
@@ -257,9 +256,10 @@ impl Drop for ProgressState {
     }
 }
 
+#[derive(Debug)]
 pub(crate) struct MultiProgressState {
     /// The collection of states corresponding to progress bars
```

Comment:
Can you add some discussion here about how `None` is used both at the start and after it's finished?